### PR TITLE
MoveAndResize and ReplayAnimation samples

### PR DESCRIPTION
**Pull request by edmaust**

### CODE REVIEW

File: Samples-Typescript/ReplayAnimation/replayanimation.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    }
+  </style>
+  <head>
+    <title>Replay Animation Demo</title>
+
+    <!-- jQuery -->
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+
+    <!-- React -->
+    <script src="https://unpkg.com/react@17/umd/react.production.min.js"></script>
```
Comment (b-carver) [inline Samples-Typescript/ReplayAnimation/replayanimation.html:16]: You shouldn't need to add React here (only for Typescript samples)

**Review by b-carver — CHANGES_REQUESTED** [review]
Can you change to merge against "dev" instead of "main"?

File: Samples-Typescript/MoveAndResize/MoveAndResize.trex [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest manifest-version="0.1" xmlns="http://www.tableau.com/xml/extension_manifest">
+  <dashboard-extension id="com.tableau.extensions.samples.moveAndResize" extension-version="0.6.0">
```
Comment (johnDance) [inline Samples-Typescript/MoveAndResize/MoveAndResize.trex:3]: Why not make this 1.0?

File: Samples-Typescript/MoveAndResize/moveAndResize.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+  interface IDashboardObjectPositionAndSizeUpdate {
+    dashboardObjectID: number;
+    height: number;
+    width: number;
+    y: number;
+    x: number;
+  }
+
+  class MoveAndResize extends React.Component {
```
Comment (johnDance) [inline Samples-Typescript/MoveAndResize/moveAndResize.tsx:16]: A few comments on what this example shows would be great.

File: Samples-Typescript/ReplayAnimation/replayAnimation.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      }
+    }
+
+    private replayAnimation = () => {
+      tableau.extensions.dashboardContent.dashboard.replayAnimationAsync(this.state.speed).then(() => {
+        console.log('done');
+      });
+    }
+  }
+  console.log('Initializing Replay Animation extension.');
```
Comment (johnDance) [inline Samples-Typescript/ReplayAnimation/replayAnimation.tsx:62]: Do you need this console log and then the log from initialize also?

File: webpack.config.js [hunk 5]
```diff
@@ -33,5 +35,11 @@ module.exports = {
   optimization: {
     minimize: false
   },
-  devtool: 'inline-source-map'
+  devtool: 'inline-source-map',
+  
+  performance: {
+    hints: false,
+    maxEntrypointSize: 500000,
```
Comment (johnDance) [inline webpack.config.js:42]: Is this getting around the warning about one the sample sizes? I thought we were going to modify the sample so it wasn't so big.

**Pull request closed without merging**